docs/guides/integration-prebidjs.md

```diff
@@ -0,0 +1,193 @@
+---
+id: integration-prebid
+title: Prebid.js Integration
+sidebar_label: Prebid.js
+pagination_label: Prebid.js Integration
+description: Information about integrating with Prebid.js as part of your UID2 implementation.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+# Prebid.js Integration Guide
+
+This guide is for publishers who want to integrate with UID2 and generate [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) (advertising tokens) to be passed by Prebid.js in the RTB bid stream.
+
+UID2 provides a [Prebid.js module](https://docs.prebid.org/dev-docs/modules/userid-submodules/unified2.html) that handles generating, storing, and refreshing UID2 tokens, as well as passing these tokens to the bid stream.
+
+In order to follow this guide you will need to make changes to the HTML and JavaScript on your site. There is no server-side work required.
```

Comment:
you can rephrase the last 3 sentences above into:

UID2 provides a Prebid.js module which has the following features:

- [x] Automatic token generation 
- [x] Automatic token refresh (as long as it has not expired)
- [x] Only changes to the front end Javascript/HTML required
- [x] No server-side changes required
- [x] Passes the UID2 token to the bid stream

I just think a list is more impactful visual wise comparing to sentences

Reply:
I like the list but feel that "No server-side changes required" and "Only changes to the front end Javascript/HTML required" are different to the others and belong on their own.

How about:

> UID2 provides a [Prebid.js module](https://docs.prebid.org/dev-docs/modules/userid-submodules/unified2.html) with the following features:
> 
> - UID2 token generation.
> - Automatic UID2 token refresh.
> - Automatic storage of UID2 tokens in the browser.
> - Automatic passing of UID2 tokens to the bid stream.
> 
> In order to follow this guide you will need to make changes to the HTML and JavaScript on your site. There is no server-side work required.

Reply:
Yeh this works.